src/main/java/cn/nukkit/event/player/PlayerChatEvent.java

```diff
@@ -21,25 +21,15 @@ public static HandlerList getHandlers() {
 
     protected Set<CommandSender> recipients = new HashSet<>();
 
-    public PlayerChatEvent(Player player, String message) {
-        this(player, message, "chat.type.text", null);
-    }
-
-    public PlayerChatEvent(Player player, String message, String format, Set<CommandSender> recipients) {
+    public void sendMessage(Player player, String message) {
         this.player = player;
         this.message = message;
+	this.format = "chat.type.text";
```

Comment:
In src/main/java/cn/nukkit/event/player/PlayerChatEvent.java, line 27.
When you call to sendMessage, it would this.format = "chat.type.text";
So in my opinion, format will always be "chat.type.text" and no others, I'm not sure if I'm right, but maybe here need to check if format is changed and you can't assign a const string to it directly.